test/jasmine/tests/plots_test.js

```diff
@@ -248,7 +248,7 @@ describe('Test Plots', function() {
     describe('Plots.resize', function() {
         var gd;
 
-        beforeEach(function(done) {
+        beforeAll(function(done) {
```

Comment:
Yep, this is on purpose.

This particular `describe` creates a graph then calls `Plotly.Plots.resize`. The multiple `it` block are simply there to group the assertion in some logical way. There's no need to re-create the graph before each `it` in this case.